packages/optimizer/README.md

```diff
@@ -125,20 +131,15 @@ will be replaced with:
 https://cdn.ampproject.org/rtv/001515617716922/v0.js
 ```
 
-Versioning the AMP runtime URLs has two benefits:
-
-1. Better usage of the brower cache: versioned AMP runtime URLs are served with a longer max-age than the unversioned ones.
-2. Critical CSS can be inlined. The AMP runtime defines it's [own CSS styles](https://cdn.ampproject.org/v0.css) for AMP layouts and other things. Versioning the AMP runtime makes it possible to inline these styles, which greatly improving time to FCP as it ensures that inlined styles and imported runtime are compatible.
+Versioning the AMP runtime URLs has one main benefit: versioned AMP runtime URLs are served with a longer max-age than the unversioned ones. This means AMP pages served with versioned AMP runtime benefit from better browser caching.
 
 **Important:** when using versioned AMP runtime URLs make sure to invalidate all caches whenever a new AMP runtime is released. This is to ensure that your AMP pages always use the latest version of the AMP runtime.  
 
 You can use [amp-toolbox-runtime-version](../amp-toolbox-runtime-version) to retrieve the latest version of the AMP runtime. Here is a sample to apply the optimizations including versioning the URLs:
 
 ```
-const ampOptimiser = require('amp-toolbox-optimizer');
-
-// retrieve the latest runtime version
-const runtimeVersion = require('amp-toolbox-runtime-version');
+const ampOptimiser = require('@ampproject/toolbox-optimizer');
```

Comment:
Maybe this should be `ampOptimizer`, for consistency with the package name. (As an Australian used to British spelling, I am very reluctant to mention this…)

Reply:
lol - fixed